components/account/AccountMoreButton.vue

```diff
@@ -62,12 +88,15 @@ async function removeUserNote() {
     <template #popper>
       <NuxtLink v-if="notLocal" :to="account.url" external target="_blank">
         <CommonDropdownItem
+          is="button"
           :text="$t('menu.open_in_original_site')"
           icon="i-ri:arrow-right-up-line"
           :command="command"
+          data-glean="profile.more.open_in_original_site"
```

Comment:
Example of a link that we could do further work in order to add 'target account'-identifiers if we really wanted to